src/utils/remote-state.ts

```diff
@@ -121,10 +122,17 @@ const createDynamoDB = async (ddbClient: DynamoDB, ddbTableName: string) => {
   }
 };
 
+const getAWSAccountID = async () => {
+  const stsClient = new STS({region: "us-east-1"});
+  const { Account: accountId} = await stsClient.send(new GetCallerIdentityCommand({}));
+  return accountId
+}
+
 export const createRemoteState = async (diggerConfig: any) => {
   if (diggerConfig.remote_state !== "local") {
+    const accountId = await getAWSAccountID()
     const ddbTableName = "digger-terraform-state-lock";
-    const bucketName = "digger-terraform-state";
+    const bucketName = `digger-terraform-state-${accountId}`;
```

Comment:
This needs to be updated on trowel side generation as well I guess?

Reply:
yep, https://github.com/diggerhq/trowel/pull/54